table/tables/tables_test.go

```diff
@@ -396,7 +396,7 @@ PARTITION BY RANGE ( id ) (
 
 	// Value must locates in one partition.
 	_, err = tb.AddRecord(ts.se, types.MakeDatums(22))
-	c.Assert(table.ErrTrgInvalidCreationCtx.Equal(err), IsTrue)
+	c.Assert(table.ErrNoPartitionForGivenValue.Equal(err), IsTrue)
```

Comment:
Could we remove `ErrTrgInvalidCreationCtx`? It seems that the error is useless now.

Reply:
Okey.

Reply:
@zimulala PTAL.